File: phalcon/app/modules/admin/controller/DesktopController.php
<?php

namespace app\modules\admin\controller;

use app\common\Base;
use app\common\AdminToken;

/* 控制台 */
class DeskTopController extends Base{

  /* 构造函数 */
  function initialize(){
    parent::initialize();
    // 控制器权限
    AdminToken::urlVerify('/');
  }

  /* 首页 */
  function indexAction(){
    return self::getJSON(['code'=>0,'msg'=>'成功']);
  }

}

File: python/app/modules/admin/DesktopController.py
from app.common.Base import Base
from app.common.AdminToken import AdminToken

class DesktopController(Base) :

  # 构造函数
  def __init__(self):
    AdminToken().urlVerify('/')

  # 首页
  def index(self):
    return self.getJSON({'code':0,'msg':'成功'})